Source/hu_stuff.c

```diff
@@ -753,13 +753,22 @@ void HU_Drawer(void)
 
   plr = &players[displayplayer];         // killough 3/7/98
   // draw the automap widgets if automap is displayed
-  if (automapactive)
+  if (automapactive && !(hud_distributed && automapoverlay))
     {
       fixed_t x,y,z;   // killough 10/98:
       void AM_Coordinates(const mobj_t *, fixed_t *, fixed_t *, fixed_t *);
 
+      if (!(hud_displayed && automapoverlay))
```

Comment:
Could this somehow get moved into `HU_Ticker()` instead?

Reply:
Don't know about this.

Reply:
There is a block in `HU_Ticker()` where the level stats and level time widgets are filled. Could you move this block which adjusts the vertical position of the map title widget to that other block?